src/reducers/issue.js

```diff
@@ -26,18 +26,14 @@ export function issues(state = initialState, action) {
         error: action.error,
         loading: false,
       })
-    case types.REQUEST_ISSUE_BY_ID:
-      return Object.assign({}, state, {
-        data: action.data || [],
-        error: false,
-        loading: true,
-      })
-    case types.REQUEST_ISSUE_BY_ID_SUCCESS:
-      return Object.assign({}, state, {
-        data: action.data || [],
-        error: false,
-        loading: true,
-      })
+    case types.UPDATE_VOTE_COUNT:
+      let newState = Object.assign({}, state);
+      newState.data.map(issue => {
```

Comment:
you need to check what this function is actually doing - `map()` doesn't modify things in place